test_fms/mpp/test_domains_utility_mod.F90

```diff
@@ -37,11 +38,11 @@ module test_domains_utility_mod
 
   contains
 
-subroutine fill_coarse_data_r8(data, rotate, iadd, jadd, is_c, ie_c, js_c, je_c, nz, isd, jsd, nx, ny, &
+subroutine fill_coarse_data_r8(rdata, rotate, iadd, jadd, is_c, ie_c, js_c, je_c, nz, isd, jsd, nx, ny, &
```

Comment:
Don't see the need to change this from data to rdata as it's local to the routine.

Reply:
not sure why i did that in the first place.. its changed back now